go.mod

```diff
@@ -17,17 +17,22 @@ require (
 	github.com/apparentlymart/go-userdirs v0.0.0-20200915174352-b0c018a67c13
 	github.com/apparentlymart/go-versions v1.0.1
 	github.com/armon/circbuf v0.0.0-20190214190532-5111143e8da2
+	github.com/aws/aws-sdk-go v1.44.122
```

Comment:
I would rather not pull in aws-sdk-go v1.x.x here.  We should not include this functionality in the testing package as we don't use it.